games-roguelike/cataclysm-dda/cataclysm-dda-0.8.ebuild

```diff
@@ -0,0 +1,181 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils toolchain-funcs flag-o-matic
+
+SP_VER="2024-10-27"
+
+DESCRIPTION="A turn-based survival game set in a post-apocalyptic world"
+HOMEPAGE="https://cataclysmdda.org
+	https://github.com/CleverRaven/Cataclysm-DDA"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/CleverRaven/Cataclysm-DDA.git"
+	SLOT="${PV}"
+else
+	trarr="0ABCDEFGH"
+	MY_PV="$(ver_cut 1).${trarr:$(ver_cut 2):1}"
+	SRC_URI="https://github.com/CleverRaven/Cataclysm-DDA/archive/${MY_PV}-RELEASE.tar.gz \
+		soundpack? ( https://github.com/Fris0uman/CDDA-Soundpacks/releases/download/${SP_VER}/CC-Sounds.zip )"
```

Comment:
Rename those files to include `${PN}`.
https://devmanual.gentoo.org/ebuild-writing/variables/index.html#renaming-sources